tools/fonts/@/gootville/LICENSE.txt

```diff
@@ -0,0 +1,5 @@
+Designed by Grzegorz Pruchniakowski.
+
+This font is licensed under the SIL Open Font License (http://scripts.sil.org/OFL).
+
+The Gootville font templates based on SMuFL and Bravura, designed by Daniel Spreadbury at Steinberg.
```

Comment:
Same for the MuseJazz fonts below. 

Are they from https://github.com/musescore/MuseScore/tree/master/fonts/musejazz

If so, can we figure out which font version we are using, and append a prefix so it looks like MuseJazz_x.y.otf, where x.y is the version number?

If you cannot figure out the version number, you can use the date the font was generated or uploaded (by inspecting the github timestamp). It's good to have some suffix to indicate the version or date, so that we can upload future versions of the font file if necessary.

Reply:
I used the ones installed in my computer by musescore. Those in github are more recent ones, I will update them with the associated release numbe attached.